needrestart_info.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+#
+#
+# Description: Expose metrics from needrestart.
+#
+# This script runs needrestart in batch mode. It will never ask for input
+# and will never restart or upgrade anything.
+#
+# Dependencies: python >= 3.5, python3-prometheus-client, needrestart
+#
+# Authors: RomainMou
+
+import time
+import subprocess
+from collections import Counter
+from enum import Enum
+from prometheus_client import (
```

Comment:
```suggestion

from prometheus_client import (
```

per convention, 3rd party module imports are separated w/ an empty line from the ones from stdlib.